v2/tools/generator/internal/config/object_model_configuration.go

```diff
@@ -37,6 +37,27 @@ func NewObjectModelConfiguration() *ObjectModelConfiguration {
 	}
 }
 
+func (omc *ObjectModelConfiguration) IsGroupExpected(pkg astmodel.PackageReference) bool {
```

Comment:
Would it make more sense to call this `ContainsGroup` or `HasGroup` or something?

`IsGroupExpected` reads weird to me. Expected for what?